### ISSUE TITLE
Animation import: empties in extras / TargetNames

### ISSUE THREAD

**Issue by capnm** [KhronosGroup/glTF-Blender-IO#1904]
**Describe the bug**
Created to track the issue in Blender and Godot Importer:
+ https://projects.blender.org/blender/blender/issues/106842
+ https://github.com/godotengine/godot/issues/75969

E.g. Babylon.js, Tree.js successfully parses both files and shows the animation.

**To Reproduce**
Compare the difference in bad/ok glTF files and try to import them:
[repro.zip](https://github.com/KhronosGroup/glTF-Blender-IO/files/11209141/repro.zip)

```
...
            {
              "NORMAL": 97,
              "_w": 96,
              "_v": 95,
              "POSITION": 94
            }
          ]
        }
      ],
      "name": "rbdbulletsolver1",
      "extras": {
        "targetNames": [
          "",
          "",
          "",
 ...
        ]
      }
    }
  ],
...
```

**Comment by julienduroure** [comment]
Blender import is solved by #1902

**Comment by julienduroure** [comment]
Fixed by 09e81e2cb02d4f139496978ce1ae9463d41eba8a
